drivers/windows/file_access_windows.cpp

```diff
@@ -205,9 +205,8 @@ void FileAccessWindows::_close() {
 			}
 		}
 
+		ERR_FAIL_COND_MSG(rename_error, save_path + ": " + "Safe save failed. This may be a permissions problem, but also may happen because you are running a paranoid antivirus. If this is the case, please switch to Windows Defender or disable the 'safe save' option in editor settings. This makes it work, but increases the risk of file corruption in a crash.");
```

Comment:
```suggestion
		ERR_FAIL_COND_MSG(rename_error, vformat("Safe save failed for '%s'. This may be a permissions problem, but also may happen because you are running a paranoid antivirus. If this is the case, please switch to Windows Defender or disable the 'safe save' option in editor settings. This makes it work, but increases the risk of file corruption in a crash.", save_path));
```